melodic/distribution.yaml

```diff
@@ -2069,6 +2069,23 @@ repositories:
       url: https://github.com/OTL/cv_camera.git
       version: master
     status: maintained
+  cyglidar_d1:
+    doc:
+      type: git
+      url: https://github.com/CygLiDAR-ROS/cyglidar_d1.git
+      version: v0.1.2
+    release:
+      tags:
+        release: release/melodic/{package}/{version}
+      url: https://github.com/CygLiDAR-ROS/cyglidar_d1-release.git
+      version: 0.0.2-1
+    source:
+      test_pull_requests: true
+      type: git
+      url: https://github.com/CygLiDAR-ROS/cyglidar_d1.git
+      version: v0.1.2
+    status: maintained
+    status_description: end-of-life
```

Comment:
This is a very odd status to be putting on a new package.  I'll suggest just removing this completely.